js/window.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+// If you use the eval function indirectly, it works in the global scope rather than
+// the local scope.
+// See https://stackoverflow.com/questions/14119988/return-this-0-evalthis/14120023#14120023
+// TODO: this should be replaced with globalThis
+// when the proposal goes to stage 4
+// See https://github.com/tc39/proposal-global
+export const window = (0, eval)("this");
```

Comment:
What does this mean? `(0, eval)` ?

Reply:
it is indirect eval
https://tc39.github.io/ecma262/#sec-performeval

Reply:
Yes, it's indirect call of eval. Basically the same as the previous code. Please see the stackoverflow link above for details.

Reply:
Please add a comment.. Something like this
```
// (0, eval) is indirect eval https://tc39.github.io/ecma262/#sec-performeval
```